docs/essentials/datetime/ChoosingBetweenDateTime.md

```diff
@@ -1 +1,212 @@
+
+> **Important**
+>
+> When saving or sharing `DateTime` data, UTC should be used and the `DateTime` value's `Kind` property should be set to `DateTimeKind.Utc`.
+
+## The DateTimeOffset structure
+
+The [System.DateTimeOffse]() structure represents a date and time value, together with an offset that indicates how much that value differs from UTC. Thus, the value always unambiguously identifies a single point in time. 
+
+The `DateTimeOffset` type includes all of the functionality of the `DateTime` type along with time zone awareness. This makes it is suitable for applications that do the following: 
+
+* Uniquely and unambiguously identify a single point in time. The `DateTimeOffset` type can be used to unambiguously define the meaning of "now", to log transaction times, to log the times of system or application events, and to record file creation and modification times. 
+
+* Perform general date and time arithmetic.
+
+* Preserve multiple related times, as long as those times are stored as two separate values or as two members of a structure.
+
+> **Note**
+>
+> These uses for `DateTimeOffset` values are much more common than those for DateTime values. As a result, `DateTimeOffset` should be considered the default date and time type for application development.
+
+A `DateTimeOffset` value is not tied to a particular time zone, but can originate from any of a variety of time zones. To illustrate this, the following example lists the time zones to which a number of `DateTimeOffset` values (including a local Pacific Standard Time) can belong.
+
+```csharp
+using System;
+using System.Collections.ObjectModel;
+
+public class TimeOffsets
+{
+   public static void Main()
+   {
+      DateTime thisDate = new DateTime(2007, 3, 10, 0, 0, 0);
+      DateTime dstDate = new DateTime(2007, 6, 10, 0, 0, 0);
+      DateTimeOffset thisTime;
+
+      thisTime = new DateTimeOffset(dstDate, new TimeSpan(-7, 0, 0));
+      ShowPossibleTimeZones(thisTime);
+
+      thisTime = new DateTimeOffset(thisDate, new TimeSpan(-6, 0, 0));  
+      ShowPossibleTimeZones(thisTime);
+
+      thisTime = new DateTimeOffset(thisDate, new TimeSpan(+1, 0, 0));
+      ShowPossibleTimeZones(thisTime);
+   }
+
+   private static void ShowPossibleTimeZones(DateTimeOffset offsetTime)
+   {
+      TimeSpan offset = offsetTime.Offset;
+      ReadOnlyCollection<TimeZoneInfo> timeZones;
```

Comment:
Is there any value in your use of ReadOnlyCollection<T>, beyond List<T>? It doesn't seem so. The value of ROC is only in the case you pass it to other readers, which doesn't happen here. Also, it's better to use ImmutableCollections if you have a read-only use case.